src/load/azext_load/data_plane/load_notifications/utils.py

```diff
@@ -0,0 +1,139 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+# Disabled since logging statements were flagged as too long
+
+from azext_load.vendored_sdks.loadtesting.models import (
+    NotificationEventType,
+    TestsNotificationRule,
+    TestRunEndedNotificationEventFilter,
+    TestRunStartedNotificationEventFilter,
+    TriggerCompletedNotificationEventFilter,
+    TriggerDisabledNotificationEventFilter,
+    TestRunEndedEventCondition,
+)
+from azure.cli.core.azclierror import InvalidArgumentValueError
+from knack.log import get_logger
+
+logger = get_logger(__name__)
+
+
+def get_notification_rule_create_body(
+    action_groups=None,
+    event=None,
+    display_name=None,
+    test_ids=None,
+    all_tests=False,
+    all_events=False,
+):
+    """
+    This function creates a NotificationRule object with all the required fields for create request.
+    """
+    logger.debug("\nEvents provided: %s\n", event)
+    event_filters = _maps_events_to_filters(event)
+    if not all_tests and test_ids is None:
+        raise InvalidArgumentValueError(
+            "No tests provided for notification rule. Please provide specific tests using --test-ids option or provide --all-tests flag to select all tests."
+        )
+
+    # If all_tests is provided we ignore test_ids provided in command
+    if all_tests:
+        logger.debug("--all-tests option selected. Ignoring values in --test-ids if provided.")
+        test_ids = None
+
+    if not all_events and not event_filters:
+        raise InvalidArgumentValueError(
+            "No events provided for notification rule. Please provide specific events using --events option or provide --all-events flag to select all events."
```

Comment:
No events provided for notification rule. Please provide specific events using --event option or provide --all-events flag to select all events.